### DISCUSSION TITLE
Can I use LowLevelZeroOptimizer in Engine so that I can use zero-1 and pipeline?

### DISCUSSION THREAD

**Discussion by yhcc** [hpcaitech/ColossalAI#2259] [Community | Q&A]
As the title.

**Comment by 1SAA** [comment]
Hi @yhcc 😀, thanks for your discussion. We have not tested the compatibility yet. I suppose the answer is no. The upgrade of pipeline is in progress now, we will ensure the compatiblity when the upgrade is done.
